tests/integration/features/marks.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Setting positional marks
+
+    Background:
+        Given I open data/marks.html
+        And I run :tab-only
+
+    ## :set-mark, :jump-mark
+
+    Scenario: Setting and jumping to a local mark
+        When I run :scroll-px 5 10
+        And I run :set-mark 'a'
+        And I run :scroll-px 0 20
+        And I run :jump-mark 'a'
+        Then the page should be scrolled to 5 10
+
+    Scenario: Jumping back jumping to a particular percentage
```

Comment:
I think this should say "Jumping back *after* jumping to a particular percentage" or so?